.gitattributes

```diff
@@ -0,0 +1,4 @@
+# exclude vendored and generated files
+package.json linguist-generated
+/src/stories/assets/** linguist-vendored
+/docs/** linguist-generated
```

Comment:
Not that familiar with this, but looking at https://github.com/github/linguist/blob/master/docs/overrides.md:

```suggestion
/docs/** linguist-documentation
```

Reply:
yep looks like linguist-documentation and lingust-generated are aliases. documentation seems more appropriate in any case so I'll accept this suggestion